python/ray/_private/runtime_env/validation.py

```diff
@@ -65,184 +214,108 @@ class RuntimeEnvDict:
     """
 
     def __init__(self,
-                 runtime_env_json: dict,
-                 working_dir: Optional[str] = None):
-        # Simple dictionary with all options validated. This will always
-        # contain all supported keys; values will be set to None if
-        # unspecified. However, if all values are None this is set to {}.
```

Comment:
Should we leave some version of this comment in the docstring for ParsedRuntimeEnv?  I guess the new guarantee is that all keys present in the parsed dict have non-None values.

Reply:
Yeah good idea!